gui/velociraptor/src/themes/ncurses.css

```diff
@@ -141,6 +141,10 @@ body.ncurses {
   font-size: var(--font-size-base) !important;
 }
 
+.ncurses .ace_editor, .ace_editor * {
```

Comment:
Probably we dont want to add to this file since this is only for the ncurses theme.

The font family is already set in the theme file for example
https://github.com/Velocidex/velociraptor/blob/00cbd2ff1c210a5a7c4c88ea864feba08472e1cf/gui/velociraptor/src/themes/veloci-light.css#L45

We actually include fonts already so the browser should not need to fall back to system fonts at all. 

Can you please check if you see the same problem in all themes and then use the dev console to check what font is actually loaded at the position of the caret?